### PR TITLE
Fixes Issue #85

### PR DESCRIPTION
**Pull request by YBirhane**

### CODE REVIEW

**Comment by YBirhane** [comment]
@raskutti

File: ac1/_sources/projects/module_a.rst [hunk 1]
```diff
@@ -14,60 +14,99 @@ In this project, you will design a survey to answer a research question of your
...
-4.  Deploy your survey and collect **at least 30** responses.
+**To Do:**
 
-    -   Having trouble getting enough responses, or want more participants? Try
-        social media!
+- Form a group of no more than 4 people. (You may want to form a group with 
+  people who are in a similar major or have similar interests as this may make 
+  question selection easier.)
+- Choose a research question that asks about the relationship between two or 
+  more attributes
```
Comment (raskutti) [inline ac1/_sources/projects/module_a.rst]: All bullet points should end with a period/question etc. A few are missing that, this one and one above. `- What percent of people are left-handed? (This question only involves one attribute)`

**Review by raskutti — APPROVED** [review]
Just one comment

**Comment by YBirhane** [comment]
Everything should be resolved! I also added the link to our example project. Anyone should be able to access the link and it is under our googleAC email.

**Comment by YBirhane** [comment]
@raskutti

**Review by raskutti — APPROVED** [review]
Nice work!

**Pull request merged**